source/clear-linux/guides/maintenance/mixer.rst

```diff
@@ -1,871 +1,865 @@
+   will execute the mixer commands from. Each mix will use its own separate
+   workspace.
 
-      sudo cp -f /usr/share/nginx/conf/nginx.conf.example /etc/nginx/nginx.conf
+#. Initialize the workspace and mix.
 
-#. Open ``mixer.conf`` with an editor.
+   Before you create a mix, you must explicitley initialize the mixer workspace.
+   During initialization, the mixer workspace is configured and the base for
+   your mix is defined. By default, your mix will be based on the latest
+   upstream version and start with the minimum set of bundles. Your first custom
+   mix version number will start at 10. You can alternately select other
+   versions or bundle sets to start from.
 
-   .. code-block:: bash
- 
-      sudo nano /etc/nginx/conf.d/mixer.conf 
+   Initialization creates the directory structure used by mixer and adds the
+   :file:`builder.conf` file, which is used to configure the mixer tool.
 
-#. Add the server configuration content below to ``mixer.conf``. 
+   View the `mixer.init man page`_ for more information on mixer
+   initialization.
 
-   .. code-block:: console
+   View the list of `suitable versions`_ to mix from.
 
-      server {
-           server_name localhost;
-           location / {
-                     root /var/www/mixer;
-                     autoindex on; 
-           }
-      }
+#. Edit builder.conf.
 
-#. Restart the daemon, enable nginx on boot, and start the service.
+   :file:`builder.conf` is used to configure the mixer tool. For example, it
+   allows you to configure where mixer output is located and where swupd update
+   content will be located.
 
-   .. code-block:: bash
+   At minimum, set the URL of your update server so your custom OS knows where
+   to get update content.
 
-      sudo systemctl daemon-reload 
+   Refer to the `builder.conf`_ section for more information.
 
-      sudo systemctl enable nginx 
+Create a mix
+============
 
-      sudo systemctl start nginx 
+A mix is created with the following steps:
 
-#. To verify the web server is running, check it in an Internet browser:
+#. Add custom RPMs and set up local repo (optional).
 
-   .. code-block:: bash
+   If you are adding custom RPMs to your mix, you will need to add the RPMs to
+   your mix workspace and set up a corresponding local repository.
 
-      http://localhost
+   You can use :ref:`autospec<autospec>`, `mock`, `rpmbuild`, or similar tools
+   to build RPMs from scratch. If the RPMs are not built on |CL|, make sure your
+   configuration and toolchain builds them correctly for |CL|. Otherwise there
+   is no guarantee they will be compatible.
 
-#. Alternatively, verify the server is running by entering:
+   Refer to the :ref:`autospec` guide for more information on using autospec to
+   build RPMs.
 
-   .. code-block:: bash
+#. Update and build bundles.
 
-      ps aux | grep nginx 
+   Add, edit, or remove bundles that will be part of your content and then build
```

Comment:
"Add, edit, or remove bundles that will be part of your content and then build
   the updated list of bundles. mixer will automatically update the
   :file:`mixbundles` file when you update the bundles in your mix."  ---> "Add, edit, or remove bundles that will be part of your mix and then build the bundles.  mixer will automatically update the
   :file:`mixbundles` file when you update the bundles in your mix."  As far as I know, there's no building "the updated list of bundles."  Once you have your list of bundles, you do build the bundles. 

Maybe to simply, break this up into 2 steps.  1. Add, edit, or remove bundles that will be part of the mix.  2. Build the bundles.